ContextJS.js

```diff
@@ -0,0 +1,201 @@
+import * as cop from "./Layers.js";
+
+// Layerfunction proceeding
+export function proceed() {
+  return cop.proceed();
+}
```

Comment:
To save function calls (until the JIT elides them, hopefully), you might want to have a look at 16.4.3 Re-exporting in http://exploringjs.com/es6/ch_modules.html